src/ansiblelint/rules/YamllintRule.py

```diff
@@ -97,6 +97,15 @@ def matchyaml(self, file: Lintable) -> List["MatchError"]:
                         tag=p.rule,
                     )
                 )
+        else:
+            matches.append(
+                self.create_matcherror(
+                    message="yamllint.config module is not present",
+                    details="Install yamllint package so ansiblelint can invoke it.",
+                    filename="YamllintRule.py",
+                    linenumber=15,
```

Comment:
Why is this line number here? I would assume that line number is irrelevant if yamllint is not installed.

Reply:
That does not make any sense. When users run ansiblelint, they expect file names and line numbers to point to the problematic points in roles and playbooks being analyzed, not to the linter code.

Also, since yamllint is an optional component of ansiblelint, I am not sure if the scenario where it is missing needs any special treatment. Also, this would be a backward-incompatible change that would require us to bump the major version of ansiblelint.